packages/website/src/pages/index.tsx

```diff
@@ -108,7 +109,7 @@ const features: FeatureItem[] = [
 function Feature({ title, description }: FeatureItem): JSX.Element {
   return (
     <div className="col col--12 padding-vert--lg">
-      <h2 className="text--center">{title}</h2>
+      <Heading as="h2" id="title" className="text--center">{title}</Heading>
```

Comment:
I like the idea of using the theme's `Heading` component, +1!

But, it's not centered visually on desktop:

<img width="1338" alt="Screenshot of a left-aligned heading above centered text" src="https://user-images.githubusercontent.com/3335181/200865187-48f18336-ec51-49c9-9e08-b0eea4388cb1.png">

Reply:
@JoshuaKGoldberg 
If there is an installation command in the website folder, where can I find it? I tried the yarn install, but Docusaurus doesn't seem to work. Is there anything I missed?

Reply:
> If there any extra installation command in the website folder

Hmm, no, the way Yarn workspaces work, running `yarn` anywhere should be fine.

> tried the yarn install, but Docusaurus doesn't seem to work

Tip: it helps to be very specific on what you've done and what's happening. By _"doesn't seem to work"_ you're not giving much information. Someone trying to help (me!) can't be specific in diagnosing your issue. Help us help you!

It also helps to say what you've read through. Have you seen the docs at https://github.com/typescript-eslint/typescript-eslint/blob/75dcdf164d206c5530ba7cc095c4599ec90abe35/DEVELOPMENT.md? What happens when you go through them?

Reply:
@JoshuaKGoldberg  I complete that the installation is finished with the yarn install command. When I execute the yarn build command, I get the same error as shown in the picture.
![build error](https://user-images.githubusercontent.com/53456037/200880095-8b6cb2eb-c33f-44b1-ad15-f3943aa62436.jpg)

Reply:
Did you run `yarn install` at the root of the repo?